azure/pom.xml

```diff
@@ -193,6 +193,11 @@
       <artifactId>commons-io</artifactId>
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <groupId>com.microsoft.azure</groupId>
+      <artifactId>azure-mgmt-batchai</artifactId>
+      <version>1.5.2-beta-SNAPSHOT</version>
```

Comment:
We usually version module as `beta` only if we are going to publish just the generated code, here we have fluent interface for batch ai service so we should version it as `1.5.2-SNAPSHOT`